BigCommerce/Integration/01_ManageBigCommerceConnection.md

```diff
@@ -0,0 +1,195 @@
+# Manage BigCommerce connection
+
+*Actindo* features a special installation app in the *BigCommerce App Store*. The *Actindo Core1 DOP connector* app setups and authorizes a connection automatically. 
+Note, if possible, you should not delete the app in *BigCommerce* after creating the connection. Although the deleting will not delete the connection in *Omni-Channel* itself, but it will invalidate the access token you have created during connecting the app. If the app was deleted by mistake, either reconnect the app or enter a valid access token manually to enable the connection again.
+
+Note that you can use the *Actindo Core1 DOP connector* app for one connection only. If you have more than one store in *BigCommerce*, you can <!--- can oder must?-->create the first connection automatically with the *Actindo Core1 DOP connector* app. For all other stores, you must create the connection manually.
+
+
+## Create BigCommerce connection automatically 
+
+Create the connection using the *BigCommerce* driver. You will not be able to configure any other settings until the connection is established.
+
+
+### Prerequisites
+
+- You have a *BigCommerce* customer account to log in to *BigCommerce*.
+- You have created a *BigCommerce* store. 
+- In the *Actindo Core1 Platform*, you have permission to create connections in *Omni-Channel*.
+
+
+### Procedure
+
+1. Log in to your *Actindo Core1* instance, on which you run the *Omni-Channel* module. To verify the successful creation later, it is important to be logged in.
```

Comment:
```suggestion
1. Log in to your *Actindo Core1* instance on which you run the *Omni-Channel* module. To verify the successful creation later, it is important to be logged in.
```
Ich bin der Meinung, da braucht man kein Komma, da es sich um eine notwendige Info handelt (s. defining vs. non-defining relative clauses: https://dictionary.cambridge.org/grammar/british-grammar/relative-clauses-defining-and-non-defining)

Reply:
ok